app/src/js/utils/dropdown-utils.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { components } from 'react-select';
+import colorVariables from '../../css/utils/variables/_colors.scss';
+import shadowVariables from '../../css/utils/variables/_shadows.scss';
+
+const { borderGrey, hoverBlue, oceanBlue, white } = colorVariables;
+const { shadowDefault } = shadowVariables;
+
+const DropdownIndicator = (props) => (
+  <components.DropdownIndicator {...props}>
+    <div className="dropdown__indicator" />
+  </components.DropdownIndicator>
+);
+
+const border = `1px solid ${borderGrey}`;
+const borderRadius = '0.5em';
+const boxShadow = shadowDefault;
+
+const customStyles = {
+  control: (base) => ({
+    ...base,
+    border,
+    borderRadius,
+    boxShadow,
+  }),
+  menu: (base) => ({
+    ...base,
+    width: 100,
```

Comment:
Not something you did, but can we get rid of this width? Causing some issues with the async dropdown menu